crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -115,7 +115,8 @@ impl DynamicScene {
 
         // Updates references to entities in the scene to entities in the world
         for (type_id, entities) in scene_mappings.into_iter() {
-            let registration = type_registry.get(type_id).unwrap();
+            let registration = type_registry.get(type_id)
+                .expect("This TypeRegistration was where we got the TypeId initially, should be safe to unwrap");
```

Comment:
One last minor nitpick. In Rust panic messages [usually written differently](https://doc.rust-lang.org/std/error/index.html#common-message-styles).
```suggestion
                .expect("this TypeRegistration was where we got the TypeId initially, should be safe to unwrap");
```